shopBuddy/shopBuddy/AppState.m

```diff
@@ -22,18 +22,30 @@ + (id)sharedManager {
 - (id)init {
     if (self = [super init]) {
         self.lists = [NSArray new];
+        self.trips = [NSArray new];
     }
     return self;
 }
 - (void)addItemToList:(ShoppingList *)list withItem: (Item *)item withCompletion:(void(^)(BOOL succeeded, NSError *error))completion{
     [ShoppingList createFromList:list withItem:item withCompletion:^(ShoppingList* updatedList, NSError *error) {
         if(!error) {
-            [ShoppingList fetchListsByUser:[PFUser currentUser] withCompletion:^(NSArray<ShoppingList *> *lists, NSError *error) {}];
-            completion(YES,nil);
+            [ShoppingList fetchLists:^(NSArray<ShoppingList *> *lists, NSError *error) {
+                completion(YES,nil);
+            }];
         }
         else {
             completion(NO,error);
         }
     }];
 }
+
+- (void)updateAppState:(void(^)(BOOL succeeded))completion {
```

Comment:
What's the purpose of this function? Could you add a comment above it explaining what it does?